packages/flutter/lib/src/rendering/binding.dart

```diff
@@ -102,7 +102,7 @@ abstract class RendererBinding extends BindingBase with ServicesBinding, Schedul
 
     registerSignalServiceExtension(
       name: 'debugDumpSemanticsTreeInTraversalOrder',
```

Comment:
debugDumpSemanticsTreeInTraversalOrder -> debugDumpSemanticsTreeInGeometricOrder

Reply:
Sorry, I did that, but forgot to upload the changes...